definitions/procedures/foreman/remove_corrupted_host_statuses.rb

```diff
@@ -0,0 +1,43 @@
+module Procedures::Foreman
```

Comment:
This is not included in any scenarios - where are you wanting it to run?

Reply:
Yes. Actually I was unsure if we want to have this to run on every upgrade or backup or other places. 
The procedures can be called as a single routine and I think it make sense to have a collection of these "maintenance" routines which can solve issues. What are your thougths?